adding disclaimer

mhant opened the pull request:
added disclaimer popup which shows until closed and then won't show again for same device

Commit:
adding disclaimer

added disclaimer popup which shows until closed and then won't show again for same device

spledgoldstein reviewed (commented):
i thought we were going to try for two buttons one to "close" for one time, and a second for "don't show again", but it looks (from the code) that there is only one button to close forever. Also, you have the show_disc stuff in the init function, but every time (e.g. if you change the kosher type filter). perhaps it should be below in the if (!filter_type), so it is only once per site opening.

Commit:
disclaimer nits

fixing so disclaimer doesn't blink with show->hide
moving disclaimer code so only called on document ready

mhant commented:
yeah two buttons looked bad with this UI component which seemed like the best fit, looks really natural as a bootstrap alert.

Good call on the code placement, forgot we called init more than once (we should probably refactor that name). Moved the code and fixed some other nits in the above commit.

Also can reset state by calling `localStorage.setItem("showDisclaimer", "yes");` in inspect console.

Commit:
title fix

fixing title of page to better fit iphone SE size phones
